### DISCUSSION TITLE
Cann't create database free plan

### DISCUSSION THREAD

**Discussion by nyeinminhtet** [planetscale/discussion#510] [FAQs] [answered]
I can not press create button why? I use free plan.

**Comment by jgreet** [comment, marked as answer]
We recently began requiring a valid credit card for all database creation, even at the free tier, as a measure to prevent fraud and abuse of our services. 

~~If you don't have a card to put down I'd suggest opening a support ticket and we can probably work something else out.~~

**Update 08-28-23**: a valid credit or debit card is now a hard requirement, we will no longer be making exemptions on a case by case basis

**Reply by ThimoDEV** [reply]
I just encountered this too. I submitted a support ticket, but I would love to have an additional form of fraud detection. I do have a virtual credit card for online services, but that was blocked. Is maybe something like a normal bank card or paypal also possible?

**Reply by jgreet** [reply]
I believe debit/bank cards should work. Prepaid virtual credit cards will not though. Paypal is an interesting idea!

**Reply by ThimoDEV** [reply]
Is it an idea to have an advanced way of kaptcha, passport records, maybe like a 1 cent payment check (ideal preferred for my location)? I want to hard separate my databases for work and private. Your support helped me out a lot to get me started, but having an extra way of verification would really help to split responsibilities.

**Reply by aviral17** [reply]
Sir, Debit card of Visa/Discover/Mastercard not being accepted, its showing your card is declined,  why is it so???? I do have balance and I want to use it for Free Tier, its not accepting why?? Please help me  urgently!!

**Reply by aviral17** [reply]
I still didnt got any answer as to why the Debit cards of Visa/Discover/MasterCard are getting declined for Free Tier

**Reply by alokshukla630** [reply]
when i interred my card details it is not showing add to card or submit button or option

**Reply by kukutulu** [reply]
I create a new organization and try to add my Visa debit card but its said "This card is already in use". Is that mean I have to add different card for each organization?

**Reply by jgreet** [reply]
If you have used the card to pay an invoice for the existing organization it should work with a new organization as well. 

Otherwise, yes, you'll need to put down different cards.

**Reply by devhimanshuhc** [reply]
can i use debit card? I dont have a credit card but a normal debit card

**Reply by jgreet** [reply]
Debit cards should work, with the exception of prepaid debit cards:
https://planetscale.com/docs/concepts/billing#payment-methods

If your particular card isn't working you could try [contacting PlanetScale support](https://support.planetscale.com/hc/en-us/requests/new) but we generally receive very little information about failing card authorizations. Often users will have to contact their bank to find out why they are declining the transaction.

**Reply by devhimanshuhc** [reply]
i have submitted a ticket though

**Comment by Feisal2023** [comment]
i am using mpesa globalpay visa card details in filling my card but i am getting feedback which says we do not accept pre-paid cards. please help me.

**Comment by sudhanshugautam2911** [comment]
I am a student and I have a simple debit card which is not supported, so what can I do? Is there any option I can verify my student ID so that I don't need to add my card?  @jgreet

**Comment by INDERJEET0202** [comment]
I am a student and I dont have any credit card to add. Is there any way that I can use planetscale database, I am creating a discord Clone and the youtuber is using planetscale database. Please give some suggession.... @jgreet

**Reply by INDERJEET0202** [reply]
And yes I got a VISA debit card from my mother, which is showing that it is not Supported. What can I do now @jgreet

**Reply by orware** [reply]
Hello @INDERJEET0202,

I've provided a response over at https://github.com/planetscale/discussion/discussions/600#discussioncomment-8251124 for you and if you have any further questions I recommend reaching out to us via https://support.planetscale.com/.

**Comment by DsChauhan08** [comment]
I am 13 year old and i don't have a credit card also my parent use debit card and don't have a credit car. How do I use? Can I use my student ID and written application from school or any other proof of identity
